templates/e-commerce/data/build.js

```diff
@@ -0,0 +1,107 @@
+const fs = require('fs');
```

Comment:
i think the output files need to be in this dir, i was expecting the products could be static , i guess you can add this to the tasks/e-commerce folder?. Let me know what you think